types/node-cron/index.d.ts

```diff
@@ -8,7 +8,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 import EventEmitter = require('events');
 
-export function schedule(cronExpression: string, func: () => void, options?: ScheduleOptions): ScheduledTask;
+export function schedule(cronExpression: string, func: (now: Date) => void, options?: ScheduleOptions): ScheduledTask;
```

Comment:
Looking at the implementation in node-cron, this date isn't required, so shouldn't this be an optional parameter?

Reply:
The parameter is always present, so it’s never undefined. You may choose not to pass it in your function though but I still think it should be like this, right?

Reply:
You're right, yes, my apologies!